packages/styles/address.css

```diff
@@ -0,0 +1,7 @@
+:root {
+  --address-line-height: 1.618;
```

Comment:
I don't know what our standards are here, but this variable doesn't seem very useful as it's only used in a single place 🤷

Reply:
This is another area where I was trying to pattern after what we do for [DescriptionList](https://github.com/dequelabs/cauldron/blob/develop/packages/styles/description-list.css#L1-L8)